### ISSUE TITLE
У дермал патча нет спрайта на мобе.

### ISSUE THREAD

**Issue by igigin** [TauCetiStation/TauCetiClassic#1028]
#### Подробное описание проблемы
Дермал патч есть, а спраета на мобе нет.

#### Что должно было произойти
Кто-то должен был его нарисовать или выпилить.

#### Что произошло на самом деле
Его не нарисовали и не выпилили.

#### Как повторить
Надеть дермал патч.

#### Дополнительная информация:
Нарисовать не могу, но могу выпилить.

**Comment by YulianR** [comment]
опять фичи выпиливаете уроды суки вам лишь бы нерфить деды старались кодили дермал патчи а вам лишь бы все обосрать вы хоть сами что-то сделалти А???

**Comment by Skippeer12** [comment]
Так ты выпилил источник серых жампсьютов.

А дермал пат без спрайта это фича, я уже создавал ишус, поищи в закрытых.

**Comment by igigin** [comment]
Говно же, хотите бронированную голову - носите шапки/шлема со спрайтами.

**Comment by ZVee** [comment]
Его и не должно быть, спрайта.
К тому же, единственная шапка, которая дает возможность иметь броню и не скрывать прическу, как во всяких там РПГ, в которых есть возможность скрыть головной убор полностью.

**Comment by igigin** [comment]
Ну а без спрайта не должно быть и вещи, либо брони на этой вещи.
Можно такую же галочку дать в преференсы, которая спраеты шапок будет выпиливать, но зачем?

**Comment by dearprincesscelestia** [comment]
Топ идея короче.

Дермал патч превращается в инпламт, который херург должен вшить в голову, и только тогда он помогает.

Или не в голову, а что там самое ценное для хеса. Может, он лысый и хочет в фуражке ходить, но зато за гроинг бережется.

**Comment by ZVee** [comment]
"Ну а без спрайта не должно быть и вещи" - могут быть исключения, единичные, собственно как сейчас. А во вторых - кто это там решил внезапно, что надо дермал патч крайними мерами менять? И на всяких беях, тг, вг тоже по сей день так и живут.

**Comment by StrikS** [comment]
Выпил фич без причины признак дурачины. :D

**Comment by Lancer411** [comment]
>невидимый имплант-броня с жуткими статами, снимаемый/надеваемый без операции
>ФИЧА

Ему либо спрайт запилить, либо операцию добавить на установку, либо хоса с ним в черепе спавнить, а с фуражки снимать броню.

**Comment by DeadOberst** [comment]
Поехавшие.
Вся суть дермал патча в отсутствии спрайта.

**Comment by RichardJones1** [comment]
Согласен, было бы реалистично, если бы спрайт был. Но тогда никто не будет носить дермал патч, так как  черный блин на лбу это ни капли не привлекательно.

**Comment by LudwigVonChesterfield** [comment]
Этот пропозал ещё имеет актуальность.

**Comment by LudwigVonChesterfield** [comment]
@TauCetiStation/sprites-department Фича или баг?

**Comment by Motzord** [comment]
Не баг а фича?

**Comment by TheLone1y** [comment]
https://media.discordapp.net/attachments/443547609777569813/888335962134941726/latex_coating_front3.gif

**Comment by Fluffy-510** [comment]
Это уже фича, предлагаю закрыть ишуй
